pages/common/2to3.md

```diff
@@ -0,0 +1,28 @@
+# 2to3
+
+> Automated Python 2 to 3 code translation.
+> More information: <https://docs.python.org/3/library/2to3.html>.
+
+- Display changes that would be performed:
+
+`2to3 {{path/to/file.py}}`
+
+- Convert a Python 2 file to Python 3:
+
+`2to3 --write {{path/to/file.py}}`
+
+- Convert a specified Python 2 feature to Python 3:
+
+`2to3 --write {{file.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Convert all features except *raw_input* the specified ones to Python 3:
```

Comment:
We use back ticks here:
```suggestion
- Convert all features except `raw_input` the specified ones to Python 3:
```

Reply:
@navarroaxel I know I saw `*...*` somewhere...

Reply:
[`pages/common/strings.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/strings.md)

Reply:
[`pages/common/supervisorctl.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/supervisorctl.md)

Reply:
@navarroaxel `strings` uses `*`, and `supervisorctl` uses `**`.  Should I use the backtick as you've done above, and open a PR to normalize those ones that I've mentioned?

Reply:
Well... We should improve our standard. Leave the * then.

Reply:
@navarroaxel should I open a PR to correct those?  I can correct the standard, too :)

Reply:
@owenvoke WDYT?

Reply:
For the `supervisorctl`, I don't think it needs the bold text (and as it's not part of the standard, it looks weird).

As for the italics in `strings`, I think we could just remove it there too, and replace the "length" with an example number. e.g.

```
- Limit results to strings that are at least 10 characters long:

`strings -n {{10}} {{file}}`
```

Reply:
>and replace the "length" with an example number

Good idea.  That must be the bit that I was missing.  Thank you!